cypress/integration/metaArticleSpec.js

```diff
@@ -0,0 +1,135 @@
+import envConfig from '../support/config/envs';
+import services from '../support/config/services';
+import testData from '../../src/app/lib/config/services';
+import {
+  getElement,
+  getSecondElement,
+  hasHtmlLangDirAttributes,
+} from '../support/bodyTestHelper';
+import {
+  checkAmpHTML,
+  checkCanonicalURL,
+  facebookMeta,
+  metadataAssertion,
+  metadataAssertionAMP,
+  openGraphMeta,
+  retrieveMetaDataContent,
+  twitterMeta,
+} from '../support/metaTestHelper';
+
+const serviceHasArticlePageType = service =>
+  services[service].pageTypes.articles !== undefined;
+
+Object.keys(services)
+  .filter(serviceHasArticlePageType)
+  .forEach(service => {
+    describe(`${service} Article Meta Tests`, () => {
+      // eslint-disable-next-line no-undef
+      beforeEach(() => {
+        cy.visit(
+          `/${service}/articles/${services[service].pageTypes.articles.asset}`,
+        );
+      });
+
+      it('should have the correct lang & dir attributes', () => {
+        hasHtmlLangDirAttributes({
+          lang: `${testData[service].datetimeLocale}`,
+          dir: `${testData[service].dir}`,
+        });
+      });
+
+      it('should have a nofollow meta tag', () => {
+        retrieveMetaDataContent('head meta[name="robots"]', 'noodp,noydir');
+      });
+
+      it('should load a maximum of two Reith font files', () => {
```

Comment:
I dont think this test should exist. All it does is assert we have no more `Reith*` font families than `ReithSans` and `ReithSerif`. The only other is the `condensed` one, we dont need a whole test (and config) to ensure it doesn't load. (and this isnt even testing if fonts load, just if the font family is used in css)